### PR TITLE
W-16539732 RPA Manager release notes 2024-08/2.

### PR DESCRIPTION
**Pull request by jhammann72**

### CODE REVIEW

File: rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc [hunk 1]
```diff
@@ -2,13 +2,47 @@
...
+
+* A new permission has been added to control the transition from test in production phase. (W-16247708)
+
+// == Fixed Issues
+include::partial$release-notes/rn-headings.adoc[tag=fixedIssues,leveloffset=+2]
+
+[%header,cols="6a,1a"]
+|===
+|Issue Resolution |ID
+| The Process Management's *Categories* page now shows only one banner. | W-14436611
```
Comment (Cristian-Venticinque) [inline rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc]:
```suggestion
| The *Categories* page in Process Management no longer shows a duplicate alert. | W-14436611
```

File: rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc [hunk 2]
```diff
@@ -2,13 +2,47 @@
...
+* A new permission has been added to control the transition from test in production phase. (W-16247708)
+
+// == Fixed Issues
+include::partial$release-notes/rn-headings.adoc[tag=fixedIssues,leveloffset=+2]
+
+[%header,cols="6a,1a"]
+|===
+|Issue Resolution |ID
+| The *Categories* page in Process Management no longer shows a duplicate alert. | W-14436611
+| Now the *Process Manager* is shown correctly in the list of *Processes for Finance Analysis*. | W-14851495
```
Comment (Cristian-Venticinque) [inline rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc]:
```suggestion
| The *Finance Analysis* view no longer shows an incorrect *Process Manager* assignment. | W-14851495
```

File: rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc [hunk 3]
```diff
@@ -2,13 +2,47 @@
...
+
+// == Fixed Issues
+include::partial$release-notes/rn-headings.adoc[tag=fixedIssues,leveloffset=+2]
+
+[%header,cols="6a,1a"]
+|===
+|Issue Resolution |ID
+| The *Categories* page in Process Management no longer shows a duplicate alert. | W-14436611
+| The *Finance Analysis* view no longer shows an incorrect *Process Manager* assignment. | W-14851495
+| During the creation of a new *Evaluation* now an input character counter is shown. | W-15301328
```
Comment (Cristian-Venticinque) [inline rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc]:
```suggestion
| *Create Evaluation* now shows an input character counter for the *Name* field. | W-15301328
```

File: rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc [hunk 4]
```diff
@@ -2,13 +2,47 @@
...
+// == Fixed Issues
+include::partial$release-notes/rn-headings.adoc[tag=fixedIssues,leveloffset=+2]
+
+[%header,cols="6a,1a"]
+|===
+|Issue Resolution |ID
+| The *Categories* page in Process Management no longer shows a duplicate alert. | W-14436611
+| The *Finance Analysis* view no longer shows an incorrect *Process Manager* assignment. | W-14851495
+| *Create Evaluation* now shows an input character counter for the *Name* field. | W-15301328
+| The assigned applications are no longer deleted during a phase change from *Build* to *Test* or from *Test* to *Production*. | W-16380363
```
Comment (Cristian-Venticinque) [inline rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc]:
```suggestion
| RPA Manager no longer deletes assigned applications during a phase change from *Build* to *Test* or from *Test* to *Production*. | W-16380363
```

File: rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc [hunk 5]
```diff
@@ -2,13 +2,47 @@
...
+include::partial$release-notes/rn-headings.adoc[tag=fixedIssues,leveloffset=+2]
+
+[%header,cols="6a,1a"]
+|===
+|Issue Resolution |ID
+| The *Categories* page in Process Management no longer shows a duplicate alert. | W-14436611
+| The *Finance Analysis* view no longer shows an incorrect *Process Manager* assignment. | W-14851495
+| *Create Evaluation* now shows an input character counter for the *Name* field. | W-15301328
+| RPA Manager no longer deletes assigned applications during a phase change from *Build* to *Test* or from *Test* to *Production*. | W-16380363
+| RPA Bots can now be updated in parallel. | W-16382151
```
Comment (Cristian-Venticinque) [inline rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc]:
```suggestion
| You can now update multiple RPA Bots in parallel. | W-16382151
```

File: rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc [hunk 6]
```diff
@@ -2,13 +2,47 @@
...
+
+[%header,cols="6a,1a"]
+|===
+|Issue Resolution |ID
+| The *Categories* page in Process Management no longer shows a duplicate alert. | W-14436611
+| The *Finance Analysis* view no longer shows an incorrect *Process Manager* assignment. | W-14851495
+| *Create Evaluation* now shows an input character counter for the *Name* field. | W-15301328
+| RPA Manager no longer deletes assigned applications during a phase change from *Build* to *Test* or from *Test* to *Production*. | W-16380363
+| You can now update multiple RPA Bots in parallel. | W-16382151
+| During the creation of an *Evaluation* a new *Category* can now be created. | W-16393605
```
Comment (Cristian-Venticinque) [inline rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc]:
```suggestion
| You can no longer create a new evaluation category with an empty name. | W-16393605
```

File: rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc [hunk 7]
```diff
@@ -2,13 +2,47 @@
...
+| It is no longer possible for multiple RPA Bots to receive the same task for executing the same run configuration. | W-16436289
```
Comment (Cristian-Venticinque) [inline rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc]:
```suggestion
| RPA Bots can no longer receive the same task for executing the same run configuration. | W-16436289
```

File: rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc [hunk 8]
```diff
@@ -2,13 +2,47 @@
...
+|===
+|Issue Resolution |ID
+| The *Categories* page in Process Management no longer shows a duplicate alert. | W-14436611
+| The *Finance Analysis* view no longer shows an incorrect *Process Manager* assignment. | W-14851495
+| *Create Evaluation* now shows an input character counter for the *Name* field. | W-15301328
+| RPA Manager no longer deletes assigned applications during a phase change from *Build* to *Test* or from *Test* to *Production*. | W-16380363
+| You can now update multiple RPA Bots in parallel. | W-16382151
+| During the creation of an *Evaluation* a new *Category* can now be created. | W-16393605
+| RPA Bots can no longer receive the same task for executing the same run configuration. | W-16436289
+| An *Alert Rule* now keeps its activation status even after a modification. | W-16528085
```
Comment (Cristian-Venticinque) [inline rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc]:
```suggestion
| Editing a deactivated *Alert Rule* no longer reactivates it. | W-16528085
```

File: rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc [hunk 9]
```diff
@@ -2,13 +2,47 @@
 
 RPA Manager is the cloud-based control plane for your automation projects.
 
+[[august-21-2024]]
+== August 21, 2024
+
+// == What's New
+include::partial$release-notes/rn-headings.adoc[tag=whatsNew,leveloffset=+2]
+
+* A new permission has been added to control the transition from test in production phase. (W-16247708)
```
Comment (Cristian-Venticinque) [inline rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc]:
```suggestion
* The new *Process publish restriction for Productionphase* permission prevents a user from moving a process from Testing to Production. 
```

File: rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc (hunk 6 above)
Comment (jhammann72) [reply rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc]: That's correct.

File: rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc [hunk 10]
```diff
@@ -2,13 +2,47 @@
...
+
+[%header,cols="6a,1a"]
+|===
+|Issue Resolution |ID
+| The *Categories* page in Process Management no longer shows a duplicate alert. | W-14436611
+| The *Finance Analysis* view no longer shows an incorrect *Process Manager* assignment. | W-14851495
+| *Create Evaluation* now shows an input character counter for the *Name* field. | W-15301328
+| RPA Manager no longer deletes assigned applications during a phase change from *Build* to *Test* or from *Test* to *Production*. | W-16380363
+| You can now update multiple RPA Bots in parallel. | W-16382151
+| During the creation of an *Evaluation* a new *Category* can now be created. | W-16393605
```
Comment (Cristian-Venticinque) [inline rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc]:
```suggestion
| You can no longer create a *Category* with an empty name. | W-16393605
```

**Review by sebermuley — APPROVED** [review]
Was there a CX Review of the permission name?

**Pull request merged**